frontend/packages/dev-console/src/utils/add-icon.ts

```diff
@@ -0,0 +1,12 @@
+import * as importGitIcon from '../../imgs/from-git.svg';
+import * as yamlIcon from '../../imgs/yaml.svg';
+import * as dockerfileIcon from '../../imgs/dockerfile.svg';
+
+const addIcons = new Map()
+  .set('from-git-icon', importGitIcon)
+  .set('yaml-icon', yamlIcon)
+  .set('dockerfile-icon', dockerfileIcon);
+
+export const getImportForIconClass = (iconClass: string): string => {
```

Comment:
Seems unnecessary to me to keep this map instead of simply importing the icons you need where you need them.